example/src/controllers/examples/subsections/basic/index.js

```diff
@@ -1,4 +1,6 @@
+const markdown = require('../../samples').markdown.basic
+const code = require('../../samples').code.basic
+
+export const content = [markdown, code];
```

Comment:
It will just let you export an arbitrary array stuff that can either be `markdown` or `<Code />` components